File: Python Code/Daily_Flash_Code_0/Week 2/25Jan/11-DailyFlash_Solutions/11-DailyFlash_Solutions/25_Jan_Solutions_Three/Python/prog3.py
num1, num2 = [int(i) for i in input("Input : ").split()]
for i in range(num1, num2 + 1):
    print("Sqr of ", i ," : ", i * i,"\ncube of ", i, " : ", i * i * i, "\n") if i % 2 == 0 else print("", end = "")

File: Python Code/Daily_Flash_Code_0/Week 2/25Jan/11-DailyFlash_Solutions/11-DailyFlash_Solutions/25_Jan_Solutions_Three/C++/prog3.cpp
#include<iostream>

int main() {
	int num1, num2;
	std::cout << "Enter num1 : ";
	std::cin >> num1;
	std::cout << "Enter num2 : ";
	std::cin >> num2;
	for(int i = num1; i <= num2; i++) {
		if(i % 2 == 0)
			std::cout << "Sqr of : " << i << " : " << i * i << "\nCube of " << i << " : " << i * i * i << "\n\n";
	}
	return 0;
}
